README.md

```diff
@@ -2,46 +2,50 @@
 
-The open-source FHIR Converter consists of the following functionality:
+Templates for HL7v2, and CCDA, to FHIR conversion come pre-installed with the Converter. HL7v2 to FHIR templates are based on the [spreadsheet](https://docs.google.com/spreadsheets/d/1PaFYPSSq4oplTvw_4OgOn6h2Bs_CMvCAU9CqC4tPBgk/edit#gid=0) created by the HL7 [2-To-FHIR project](https://confluence.hl7.org/display/OO/2-To-FHIR+Project). If needed, you can create new, or modify existing templates by following this document, and deploy those to meet your specific conversion requirements.
 
-1. A set of starting templates, leveraging [handlebars](https://handlebarsjs.com/), to translate HL7 v2 messages or CDA documents into FHIR bundles.
-1. A set of sample data to accompany the released templates.
-1. A collection of APIs to convert data real time and assist in the template management and creation.
-1. A Web UI editor to modify and create templates and test single data conversion to FHIR bundles.
+FHIR converter runs as a REST web service and can be deployed on-prem or in the cloud. It takes raw data as input and converts it to FHIR bundles. These bundles can be persisted to a FHIR server such as the [Azure API for FHIR](https://azure.microsoft.com/en-us/services/azure-api-for-fhir/). 
+
+The FHIR Converter consists of the following components:
+
+1. [Conversion APIs](docs/api-summary.md) for converting data in request-response mode
+1. Pre-installed [set of templates](src/templates) for HL7v2 and CDA to FHIR conversion.
+1. [Sample data](src/sample-data) for testing purpose.
+1. A [Browser based editor](docs/web-ui-summary.md) to modify, create, and test templates.
+1. [Template management APIs](docs/api-summary.md) to manage the templates
+1. An embedded git server as another way to manage the templates
+
+The FHIR Converter released to open source on Thursday March 6th, 2020 with support for HL7 v2 to FHIR conversion. On Friday June 12th, 2020, C-CDA to FHIR conversion was added to the OSS FHIR Converter.
 
 FHIR® is the registered trademark of HL7 and is used with the permission of HL7. Use of the FHIR trademark does not constitute endorsement of this product by HL7.
 
-## Deploy the FHIR Converter
+## Deploying the FHIR Converter
 
-The source code is available to be deployed in any manner you would like. The FHIR Converter can be run on-prem or in the cloud. To deploy the FHIR Converter, there are two key pieces of functionality. One is a web server (node.js) and the other is a persistence layer for the templates. To assist with easy deployment we have included two options below, one through Azure and one which will deploy locally. If you choose to deploy from another mechanism then the two options below, you will need to setup the storage for the templates.
+The FHIR Converter can be deployed to Azure to run locally.

```

Comment:
Maybe you want `or run locally` instead of `to run locally`?

Reply:
Yes. Fixed.